index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+	<title>Jordan Addis - Portfolio</title>
+	<!-- Styling -->
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
```

Comment:
You might want to swap normalize and your stylesheet around as the code is read top down and this can cause conflicts between what you want to style and what default styling is a part of normalize.css